ykllvmwrap/src/ykllvmwrap.cc

```diff
@@ -221,10 +222,11 @@ extern "C" void *compileModule(string TraceName, Module *M,
   // Allocate space for compiled trace address, stackmap address, and stackmap
   // size.
   // FIXME This is a temporary hack until the redesigned hot location is up.
-  uintptr_t *ptr = (uintptr_t *)malloc(sizeof(uintptr_t) * 3);
+  uintptr_t *ptr = (uintptr_t *)malloc(sizeof(uintptr_t) * 4);
   ptr[0] = EE->getFunctionAddress(TraceName);
   ptr[1] = (uintptr_t)SMR.Ptr;
   ptr[2] = SMR.Size;
+  ptr[3] = (uintptr_t)AOTMappingVec;
```

Comment:
You are gonna hate me, but the PR still introduces a lot of C-style casts.

Reply:
thanks.

so `static_cast` didn't work, hence `reinterpret_cast`?

Reply:
Yeah, `static_cast` isn't allowed cause we're casting a pointer to an int.